app/domain/errors.rb

```diff
@@ -365,6 +365,12 @@ module AuthnGcp
         msg:  "Role must have at least one of the following constraints: {0-constraints}",
         code: "CONJ00069E"
       )
+
+      InvalidAccountInAudience = ::Util::TrackableErrorClass.new(
+        msg:  "'audience' token claim '{0}' is invalid. " \
+              "The account in the audience '{1}' does not match the account in the URL request '{2}'",
```

Comment:
End with a period.

Reply:
I look at other messages with similar format and we do not have here a period at the end

Reply:
@shulifink

Reply:
@nessiLahav  - approved
Re eran's comment - Can you please send me a screenshot of where these appear?
Then I will be able to see if they need a full stop (period) or not

Reply:
yes i will send you in slack, thank you :)